fuzz/oss-fuzz-build.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash -eu
```

Comment:
README sounds better. However, can we wait until the deliverable report is ready? I suppose there will be more tests and adjustments after the deployment and I wanted to avoid documenting something in the semi-final form.